Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/cxf/CONNECTSamlAssertionValidator.java

```diff
@@ -139,13 +108,34 @@ protected void validateAssertion(final SamlAssertionWrapper assertion) throws WS
                 LOG.error("Saml Validation error: " + e.getMessage(), e);
                 throw new WSSecurityException(ErrorCode.FAILED_CHECK, "invalidSAMLsecurity");
             }
+        }
+    }
+
+    /**
+     * @param issuer
+     * @throws WSSecurityException
+     */
+    private static void validateIssuer(Issuer issuer) throws WSSecurityException {
+        if (SamlConstants.URN_OASIS_NAMES.equals(issuer.getFormat())) {
+            if (!StringUtils.isBlank(issuer.getSPProvidedID())) {
+                throw new WSSecurityException(ErrorCode.FAILED_CHECK,
+                    SamlConstants.SECURITY_ASSERTION_ISSUER_FORMAT + "" + SamlConstants.URN_OASIS_NAMES + "" + "and"
```

Comment:
Should there be a space within the open/close quotes?  Otherwise, there are unnecessary string concatenations within this statement.